chatbot.py

```diff
@@ -487,14 +518,20 @@ def process_msg(
 TWILIO_AUTH_TOKEN: str = os.getenv(
     "TWILIO_AUTH_TOKEN")  # type: ignore [assignment]
 TWILIO_NUMBER: str = os.getenv("TWILIO_NUMBER")  # type: ignore [assignment]
-SUBSCRIBER_FILE: str = "bot_subscribers/team56test.json"
+SUBSCRIBER_FILE: str = "bot_subscribers/subscribers.json"
 BACKUP_FILE: str = "bot_subscribers/backup.json"
 KEY_FILE: str = "json/key.key"
+LOGS_FILE: str = "json/logs.json"
+BACKUP_LOGS_FILE: str = "json/backup_logs.json"
+LOGS_KEY_FILE: str = "json/key2.key"
 mr_botty = Chatbot(
```

Comment:
BTW, you don't need to add keyword arguments to this because they have defaults we're using.